PWGHF/correlationHF/AliAnalysisTaskSELambdacCorrelations.cxx

```diff
@@ -1832,6 +1837,21 @@ void AliAnalysisTaskSELambdacCorrelations::CreateCorrelationsObjs() {
  
     //both for SE and for ME
     
+    //D* feeddown pions rejection histos
+    namePlot = "hSigmacPionsVsDmass_Bin"; namePlot+=i;
+    TH2F *hDstarPions = new TH2F(namePlot.Data(), "Tracks rejected for D* inv.mass cut vs D inv mass; # Tracks",2,0.,2.,150,1.5848,2.1848);
```

Comment:
This needs to be adapted to the correct invariant mass range for the Lambdac

Also, please update hDstarPions/hDstarPions2 names with Sigmac instead of Dstar